blog/2024-11-20-pytorch-channel-deprecation.md

```diff
@@ -0,0 +1,75 @@
+---
+authors:
+  - core
+tags: [pytorch]
+---
+
+# About Pytorch Channel Deprecation
+
+[PyTorch team announced](https://github.com/pytorch/pytorch/issues/138506) that they will stop publishing packages to their official channel on Anaconda.org (`-c pytorch`) after the `2.5` release. We want to address this change and clarify our commitment to maintaining PyTorch packages on conda-forge. 
+
+First and foremost, we deeply appreciate the PyTorch team’s significant contributions to the conda packaging ecosystem and their ongoing collaboration with conda-forge. The conda-forge team sees this transition as an opportunity to strengthen the conda ecosystem and create more stable and reliable environments for users.
+
+<!-- truncate -->
+
+Dependency conflicts have long been a pain point for users when mixing packages from the conda-forge and Anaconda "defaults." These conflicts arise because different channels build packages against different versions of core dependencies (packages on pytorch channel had dependancies in Anaconda `defaults` cannels). By consolidating PyTorch packages entirely under conda-forge, we can provide a more consistent and reliable experience, reducing conflicts and improving installation stability.
+
+
+## Our Commitment
+
+- Conda-forge is actively working with PyTorch maintainers and the broader open-source community to address potential gaps, ensure stability, and improve package availability.
```

Comment:
I’m not sure that this would be my first commitment or any commitment at all on the list. 

My only commitment is to high quality packaging and working in the open. 

If and when those involved in the upstream pytorch package want to get involved in the conda-forge effort then I am committed to review it based on my availability as a volunteer.